thunder/benchmarks/benchmark_litgpt.py

```diff
@@ -496,3 +496,7 @@ def benchmark_main(return_metrics_as_json=False, json_path="", **kwargs) -> None
     from jsonargparse import CLI
 
     CLI(benchmark_main)
+
+    # ref: https://github.com/pytorch/pytorch/blame/3af12447f85dfede191a113c052e58fa7b21a8b3/torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp#L1110-L1116
```

Comment:
nit: shorter link that is not a git blame

```suggestion
    # ref: https://github.com/pytorch/pytorch/blob/3af12447/torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp#L1110-L1116
```